src/components/Home/Buttons.jsx

```diff
@@ -37,3 +49,18 @@ export default () => (
     </div>
   </ButtonsDiv>
 );
+
+Buttons.propTypes = {
+  user: PropTypes.shape({}),
+};
+
+class RequestLayer extends React.Component {
```

Comment:
is there any reason to have this requestlayer here?  you should be able to connect the buttons component directly.